src/store/user.ts

```diff
@@ -9,7 +9,7 @@ import type { User } from 'src/services/renown/types';
 
 let userInit = false;
 
-const userAtom = atom<User | undefined>(undefined);
+const userAtom = atom<User | null | undefined>(undefined);
```

Comment:
The user starts as undefined, when the initial check is finished and no user is found then it's set to null.